system/Boot.php

```diff
@@ -149,6 +149,9 @@ public static function bootTest(Paths $paths): void
         static::setExceptionHandler();
         static::initializeKint();
         static::autoloadHelpers();
+
+        // Global test helpers and mocks for all tests
+        require_once ROOTPATH . 'tests/_support/MockCommon.php';
```

Comment:
I was thinking about using this one: https://github.com/codeigniter4/CodeIgniter4/blob/4.7/system/Test/Mock/MockCommon.php

You can create an additional method like:
```php
protected static function loadCommonFunctionsMock(): void
{
    // Require system/Test/Mock/MockCommon.php
    if (is_file(SYSTEMPATH . 'Test/Mock/MockCommon.php')) {
        require_once SYSTEMPATH . 'Test/Mock/MockCommon.php';
    }
}
```

and call it just before `static::loadCommonFunctions()`, so that our cli mock has a chance to be loaded.